core/src/com/unciv/models/metadata/ModCategories.kt

```diff
@@ -81,6 +82,6 @@ class ModCategories : ArrayList<ModCategories.Category>() {
             }
         }
         save()
-        return "$newCount new categories"
+        return "${newCount.tr()} new categories"
```

Comment:
Unnecessary, this is for devs only - and the rest of the line isn't translated anyway